subworkflows/local/utils_nfcore_callingcards_pipeline/main.nf

```diff
@@ -0,0 +1,316 @@
+    // Can use ternary operators to dynamically construct based conditions, e.g. params["run_xyz"] ? "Tool (Foo et al. 2023)" : "",
+    // Uncomment function in methodsDescriptionText to render in MultiQC report
+    def citation_text = [
+            "Tools used in the workflow included:",
+            "bedtools (Quilan et all 2010)",
+            "bowtie (Langmead et al. 2009)",
+            "bowtie2 (Langmead et al. 2012)",
+            "bwa (Li and Durbin 2009)",
+            "bwamem2 (Vasimuddin Md et all 2019)",
+            "callingCardsToools (Mateusiak 2023)",
+            "FastQC (Andrews 2010),",
+            "MultiQC (Ewels et al. 2016)",
+            "picard (Broad Institute 2019)",
+            "rseqc (Wang et al. 2012)",
+            "samtools (Danecek et all 2021)",
+            "seqkit (Shen et all 2016)",
+            "trimmomatic (Bolger et all 2014)",
+            "UMITools (Smith et al 2017)",
+            "."
+        ].join(' ').trim()
+
+    return citation_text
+}
+
+def toolBibliographyText() {
+    // Can use ternary operators to dynamically construct based conditions, e.g. params["run_xyz"] ? "<li>Author (2023) Pub name, Journal, DOI</li>" : "",
+    // Uncomment function in methodsDescriptionText to render in MultiQC report
+    def reference_text = [
+            "<li>Aaron R. Quinlan, Ira M. Hall, BEDTools: a flexible suite of utilities for comparing genomic features, Bioinformatics, Volume 26, Issue 6, March 2010, Pages 841–842, https://doi.org/10.1093/bioinformatics/btq033</li>",
+            "<li>Langmead B, Trapnell C, Pop M, Salzberg SL. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol 10:R25.</li>",
+            "<li>Langmead B, Salzberg S. Fast gapped-read alignment with Bowtie 2. Nature Methods. 2012, 9:357-359.</li>",
+            "<li>Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics. 2009 Jul 15;25(14):1754-60. doi: 10.1093/bioinformatics/btp324. Epub 2009 May 18. PMID: 19451168; PMCID: PMC2705234.</li>",
+            "<li>Vasimuddin Md, Sanchit Misra, Heng Li, Srinivas Aluru. Efficient Architecture-Aware Acceleration of BWA-MEM for Multicore Systems. IEEE Parallel and Distributed Processing Symposium (IPDPS), 2019. 10.1109/IPDPS.2019.00041</li>",
+            "<li>Chase Mateusiak. (2023). cmatKhan/callingCardsTools: v1.2.0 (v1.2.0). Zenodo. https://doi.org/10.5281/zenodo.10042067</li>",
+            "<li>Andrews S, (2010) FastQC, URL: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/).</li>",
+            "<li>Ewels, P., Magnusson, M., Lundin, S., & Käller, M. (2016). MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics , 32(19), 3047–3048. doi: /10.1093/bioinformatics/btw354</li>",
+            "<li>“Picard Toolkit.” 2019. Broad Institute, GitHub Repository. https://broadinstitute.github.io/picard/; Broad Institute.</li>",
```

Comment:
Different structure to the other references?

Reply:
I just followed the example in the template, I believe. I'm not actually sure where this prints out to check it

Reply:
As an aside, it seems like the tools and bibliography are redundant -- maybe just the bib is good enough?